src/parse_arx.cc

```diff
@@ -28,22 +28,20 @@
 #include <iostream>
 #include <lttoolbox/xml_parse_util.h>
 
-void print_tags(vector<wstring> input)
+void print_tags(vector<UString> input)
```

Comment:
We should check everywhere vectors are passed by value. That's needlessly expensive.